sdk/python/feast/utils.py

```diff
@@ -256,3 +256,13 @@ def _convert_arrow_to_proto(
         created_timestamps = [None] * table.num_rows
 
     return list(zip(entity_keys, features, event_timestamps, created_timestamps))
+
+
+def tags_str_to_dict(tags: str):
```

Comment:
```suggestion
def tags_str_to_dict(tags: str) -> Dict[str, str]:
```

Reply:
It'd be good to have an explicit test for this.

Reply:
done